pkg/config/config_template.yaml

```diff
@@ -701,6 +701,15 @@ api_key:
 #
 # secret_backend_skip_checks: false
 
+## @param secret_backend_command_sha256 - string - optional
+## @env DD_SECRET_BACKEND_COMMAND_SHA256 - string - optional
+## `secret_backend_command_sha256` is the SHA256 of the backend command executable.
```

Comment:
WDYT ?
```suggestion
## `secret_backend_command_sha256` is the expected SHA256 of the backend command executable. The agent will refuse to start if the SHA256 of 'secret_backend_command' is different from the one provided here.
```

Reply:
Sounds good, I will update this description.